src/fastoad/openmdao/variables.py

```diff
@@ -357,13 +358,21 @@ def update(self, other_var_list: "VariableList", add_variables: bool = True):
               in other_var_list
             - if not, Variable instance from other_var_list will be added only if
               add_variables==True
+            - The Variable instance description property is replaced by the property of the
+              the Variable instance in other_var_list if this latter is not empty.
+            - Otherwise, the description property is kept from the current Variable instance.
```

Comment:
Looks overly complicated to me. After the doc tells about the case where the Variable will be replaced, then about the case where the Variable will be added, you put an additional information that applies when the Variable will be replaced.

I think it can be simpler and clearer by just putting the additional information at the right place:
```suggestion
              in other_var_list (special case: if the one in other_var_list has an empty description, the original description is kept)
            - if not, Variable instance from other_var_list will be added only if
              add_variables==True
```

Reply:
Correction has been done

Reply:
The branch has been rebased on current master